test-suite/output/ltac2_match_warning_14135.out

```diff
@@ -0,0 +1,4 @@
+File "./output/ltac2_match_warning_14135.v", line 19, characters 4-10:
+Warning: The following clause is redundant. [redundant-clause,ltac]
+File "./output/ltac2_match_warning_14135.v", line 26, characters 4-5:
+Warning: The following clause is redundant. [redundant-clause,ltac]
```

Comment:
Maybe we should generalize warn_redundant_clause to also be able to specify `ltac2` instead of `ltac` here.

Also it seems strange too say "The following clause is redundant" since nothing follows the message. Perhaps it would be better to change the wording to "This clause is redundant". @coq/doc-maintainers Let me know what you think about this. It could also come in another PR.

Reply:
I'd change the wording in this PR since it's so easy to so.  Maybe "Redundant branch in match."?

Reply:
> Maybe we should generalize warn_redundant_clause to also be able to specify ltac2 instead of ltac here.

No generalization needed, it's a Ltac2-specific warning, we can simply introduce a "ltac2" warning category